Gemfile

```diff
@@ -43,6 +43,10 @@ gem 'rails-observers', '~> 0.1.2'
 gem 'ckeditor', github: 'galetahub/ckeditor'
 gem 'autoprefixer-rails'
 
+gem 'oauth2', '1.0.0'
+gem 'google-api-client', '0.7.1'
+gem 'legato', '0.4.0'
```

Comment:
Do you need these gems to be exactly in specified versions?

In other words, would it break if I updated google-api-client to a more recent version?